pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -58,7 +58,7 @@ func BuildKubernetesMasterConfig(options configapi.MasterConfig, requestContextM
 	portalNet := net.IPNet(flagtypes.DefaultIPNet(options.KubernetesMasterConfig.ServicesSubnet))
 
 	// in-order list of plug-ins that should intercept admission decisions
-	admissionControlPluginNames := []string{"NamespaceExists", "NamespaceLifecycle", "LimitRanger", "ResourceQuota"}
+	admissionControlPluginNames := []string{"NamespaceExists", "NamespaceLifecycle", "OriginPodNodeEnvironment", "LimitRanger", "ResourceQuota"}
```

Comment:
Per our previous discussion, we need to track this as an example of a feature that is lost when we look to support running OpenShift master as a pod on an existing Kubernetes cluster.  Can you leave a TODO noting that OriginPodNodeEnvironment is not upstream, and we would need something similar to support that node-selector default for a namespace upstream?